### PR TITLE
Adding 3.0 to list of subscriptions in order to mirror these branches

### PR DESCRIPTION
**Pull request by Anipik**
Corert does not have any release branches so removing them from the list of subscriptions

### CODE REVIEW

**Comment by Anipik** [comment]
cc @danmosemsft  @eerhardt @safern

**Comment by danmosemsft** [comment]
This governs which branches export and import which artefacts doesn't it? Don't the 3.0 branches already consume each other?

**Comment by danmosemsft** [comment]
cc @mmitche

**Comment by Anipik** [comment]
> This governs which branches export and import which artefacts doesn't it?

All the items in this .json file are independent  of each other. This item is mirror specific and controls the commtis being added to the database. if the paths modified in the webhook contains any of the trigger paths, the vsts build definition specified on https://github.com/dotnet/versions/pull/485/files#diff-8a1f1781ca237736e692e7be485c34abR2073 with be triggered with appropriate arguments.

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -2064,10 +2064,11 @@
       "triggerPaths": [
         "https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/**/*",
         "https://github.com/dotnet/corefx/blob/release/2.1/src/Common/src/CoreLib/**/*",
+        "https://github.com/dotnet/corefx/blob/release/3.0/src/Common/src/CoreLib/**/*",
         "https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/**/*",
         "https://github.com/dotnet/coreclr/blob/release/2.1/src/System.Private.CoreLib/shared/**/*",
-        "https://github.com/dotnet/corert/blob/master/src/System.Private.CoreLib/shared/**/*",
-        "https://github.com/dotnet/corert/blob/release/2.1/src/System.Private.CoreLib/shared/**/*"
```
Comment (eerhardt) [inline Maestro/subscriptions.json]: Why are we removing this line?

`https://github.com/dotnet/corert/blob/release/2.1/src/System.Private.CoreLib/shared/**/*`

Comment (eerhardt) [reply Maestro/subscriptions.json]: Oh, I just read why in your original description.

Comment (Anipik) [reply Maestro/subscriptions.json]: > Note, you don't need to redeploy Maestro to make updates to the subscriptions.json file. Merging a PR that changes the master branch will automatically be picked up by the Maestro service.

Thanks.

> Why are we removing this line?

The release branches are not present in corert repo. so this link was just broken

**Pull request merged**